doc/guides/4 - Refinery Extensions/3 - Testing.textile

```diff
@@ -26,6 +26,12 @@ And run
 $ bundle install
 </ruby>
 
+You will need to install phantomjs from http://phantomjs.org/ or through your OS package manager.
+
+<shell>
+$ sudo apt-get install phantomjs
+</shell>
+
```

Comment:
I can do that for sure.  Is there a good way to do it without taking up 3/4 of the screen for a simple little dependency install?

![screen shot 2016-05-10 at 9 18 20 pm](https://cloud.githubusercontent.com/assets/4333395/15169892/26e4a13c-16f5-11e6-9f77-62643a23cd19.png)